.yamato/yamato-config.yml

```diff
@@ -15,14 +18,14 @@ commands:
     - command: |
         linecoverage=$(head -2 test-results/coverage.xml | grep -Eo 'line-rate="[0-9]+([.][0-9]+)?"' | grep -Eo "[0-9]+([.][0-9]+)?")
         echo "Line coverage: $linecoverage"
-        if (( $(echo "$linecoverage < 0.3" | bc -l) )); then exit 1; fi
+        if (( $(echo "$linecoverage < $COVERAGE_EXPECTED" | bc -l) ));
+        then echo "ERROR: Code Coverage is under threshold of $COVERAGE_EXPECTED" && exit 1
+        fi
 triggers:
     cancel_old_ci: true
     expression: |
-        (pull_request.target eq "main"  AND
-        NOT pull_request.push.changes.all match "**/*.md") OR
-        (pull_request.target eq "dev" AND
-        NOT pull_request.push.changes.all match "**/*.md")
+        (pull_request.target eq "ROS2" OR push.branch eq "ROS2" OR pull_request.target eq "ROS2dev" OR push.branch eq "ROS2dev")
```

Comment:
I am curious why the trigger is only on ROS2 and ROS2dev.

Reply:
In the endpoint we have two parallel workstreams.  There's main & dev, and ROS2/ROS2dev. This PR is going into ROS2dev, so it needs to use modified triggers to target the relevant branches